benchmark/httpapi.py

```diff
@@ -114,10 +137,11 @@ def post(self, request):
         request.setHeader(b'content-type', b'application/json')
         try:
             json = loads(request.content.read())
+            timestamp_parser.parse(json['timestamp'])
```

Comment:
Yea, actually now that I know this is just a little front-end for a internal-only test server I can go either way on that :)